systemvm/debian/opt/cloud/bin/diagnostics.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import shlex
+import subprocess
+import sys
+
+
+def run_cmd(command):
+    if command is not None:
+        try:
+            p = subprocess.Popen(shlex.split(command), stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+            stdout, stderr = p.communicate()
+            return_code = p.returncode
+
+        except OSError as e:
+            stdout = "Check your command type"
+            stderr = "Exception occurred: %s" % e
+            return_code = 127
+
+        finally:
+            print('%s}' % stdout.strip())
```

Comment:
Improve prefix/container for stdout and stderr with something like `STDOUT{%value here}`, `STDERR{%value here}`. Don't assume that closing brace `}` may not be part of stdout or stderr.

Reply:
Change, using && for parsing

Reply:
This leads to this: exitcode = EXITCODE: {0}

Will parae the details as {} without the preceeding STDERR/STDOUT/EXITCODE